react/README.md

```diff
@@ -450,6 +450,7 @@
     ```
 
   - Do not use underscore prefix for internal methods of a React component.
+  > Why? Underscore prefixes are sometimes used as a convention in other languages to denote privacy. But, unlike those languages, there is no native support for privacy in JavaScript, everything is public. Regardless of your intentions,  adding underscore prefixes to your properties does not actually make them private, and any property (underscore prefixed or not) should be treated as being public. See issues #1024, and #490 for a more in-depth discussion.
```

Comment:
there shouldn't be two spaces after "intentions,", and let's do "underscore-prefixed".

In what other languages is a single underscore used as a convention for privacy? I'm only aware of it in JS. PHP (and I think Python) use double underscores, but that's not the convention we're addressing here.

Otherwise, looks good!